tests/carma_sulfate_vehkamaki_test.F90

```diff
@@ -0,0 +1,380 @@
+!! This code is to test the impact of particle swelling from
```

Comment:
This description should be updated with the actual purpose of this test. This is a copy and past error that may be in several of the tests.

Reply:
fixed